sdks/go/test/integration/primitives/windowinto.go

```diff
@@ -93,8 +93,15 @@ func WindowSums_Lifted(s beam.Scope) {
 	WindowSums(s.Scope("Lifted"), stats.SumPerKey)
 }
 
-func validate(s beam.Scope, wfn *window.Fn, in beam.PCollection, tr window.Trigger, m window.AccumulationMode, expected ...interface{}) {
-	windowed := beam.WindowInto(s, wfn, in, beam.WindowTrigger{Name: tr}, beam.AccumulationMode{Mode: m})
+func validateEquals(s beam.Scope, wfn *window.Fn, in beam.PCollection, tr window.Trigger, m window.AccumulationMode, expected ...interface{}) {
+	var accumulationMode beam.AccumulationMode
+	switch m {
+	case window.Accumulating:
+		accumulationMode = beam.PanesAccumulate()
+	case window.Discarding:
+		accumulationMode = beam.PanesDiscard()
+	}
```

Comment:
Consider instead changing `m` to be `beam.AccumulationMode` and just passing in the argument as before, rather than delegating to a switch, etc.